backend/tests/Notifo.Domain.Tests/Integrations/Discord/DiscordTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Notifo.Domain.Integrations.Discord;
+public sealed class DiscordTests
+{
+    [Fact]
+    public void Should_get_integration()
+    {
+        var client = CreateClient();
+        Assert.NotNull(client.System);
+    }
+
+    [Fact]
+    public async Task Should_send_simple_message()
+    {
+        var client = CreateClient();
+
+        var message = new MessagingMessage
+        {
+            Text = "Test message",
+        };
+
+        var userId = GetUserId();
+        AddTargets(message, userId);
+
+        var result = await client.System.SendAsync(client.Context, message, default);
+
+        Assert.Equal(DeliveryResult.Handled, result);
+    }
+
+    [Fact]
+    public async Task Send_full_message()
+    {
+        var client = CreateClient();
+
+        var message = new MessagingMessage
+        {
+            Text = "Test message",
+            DetailedBodyText = "Detailed body text",
+        };
+
+        var userId = GetUserId();
+        AddTargets(message, userId);
+        var result = await client.System.SendAsync(client.Context, message, default);
+
+        Assert.Equal(DeliveryResult.Handled, result);
+    }
+
+    [Fact]
+    public async Task Should_failed_on_user_without_app_installed()
+    {
+        var client = CreateClient();
+
+        var message = new MessagingMessage { Text = "Test message" };
+
+        var userId = "261155655006000000"; // Random userId
```

Comment:
If something is random, I usually make it random with "Guid.NewGuid().ToString()"